### DISCUSSION TITLE
serilog-sinks-skywalking

### DISCUSSION THREAD

**Discussion by SpringHgui** [SkyAPM/SkyAPM-dotnet#485] [Ideas]
I wrote a plugin to support serilog logs to automatically write skywalking what the official thinks of this idea
After my testing, the log writes are as expected
Is there any drawback

[serilog-sinks-skywalking](https://github.com/SpringHgui/serilog-sinks-skywalking)

**Comment by Superskyyy** [comment]
Wow thanks, are you planning to contribute the plugin to this repository? FYI @liuhaoyang

**Comment by SpringHgui** [comment]
yes

**Comment by wu-sheng** [comment]
If so, you could create a pull request, and reviewer could start from there.

**Comment by SpringHgui** [comment]
ok，wait for my free time
